app/database_etl/etl_main.py

```diff
@@ -254,13 +251,63 @@ def get_coords(place_name, place_type):
                 break
     return coords
 
+
 def add_latlng_to_df(place_type, place_type_name, df):
     df['coords'] = df[place_type_name].map(lambda a: get_coords(a, place_type))
     df['longitude'] = df['coords'].map(lambda a: a[0] if isinstance(a, list) else None)
     df['latitude'] = df['coords'].map(lambda a: a[1] if isinstance(a, list) else None)
     df = df.drop(columns=['coords'])
     return df
 
+
+def get_most_recent_pub_date_info(row):
+    # Get index of most recent pub date if the pub date is not None
+    try:
+        pub_dates = row['publication_date']
+        max_index = pub_dates.index(max(pub_dates))
+        row['publication_date'] = row['publication_date'][max_index]
+
+    # If pub date is None set to index to 0
+    except AttributeError:
+        max_index = 0
+
+    # If source type exists, get element at that index
+    if row['source_type']:
+        row['source_type'] = row['source_type'][max_index]
+
+    # Index whether org author exists and corresponding first author
+    is_org_author = row['organizational_author'][max_index]
+    row['organizational_author'] = is_org_author
+    row['first_author'] = row['first_author'][max_index]
+
+    # If it is not an organizational author, then get last name
+    if not is_org_author:
+        row['first_author'] = row['first_author'].strip().split()[-1]
+    return row
+
+
+def apply_min_risk_of_bias(df):
+    bias_hierarchy = ['Low', 'Moderate', 'High', 'Unclear']
+    for name, subset in df.groupby('study_name'):
+        if (subset['overall_risk_of_bias']).isnull().all():
+            subset['overall_risk_of_bias'] = 'Unclear'
+            continue
+        for level in bias_hierarchy:
+            if (subset['overall_risk_of_bias'] == level).any() or level == 'Unclear':
+                subset['overall_risk_of_bias'] = level
+                continue
+    return df
+
+
+def get_city(row):
+    if row['city']:
+        return row['city'].split(',')
+    elif row['county']:
+        return row['county'].split(',')
+    else:
```

Comment:
nit: So this else case would happen if there's no city or county right? Would it be better to just return None or '' instead of returning row['city'], seems slightly confusing